lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -134,15 +132,13 @@ extension DomWindowExtension on DomWindow {
   DomMediaQueryList matchMedia(String? query) => _matchMedia(query?.toJS);
 
   @JS('getComputedStyle')
-  external DomCSSStyleDeclaration _getComputedStyle1(DomElement elt);
-  @JS('getComputedStyle')
-  external DomCSSStyleDeclaration _getComputedStyle2(
-      DomElement elt, JSString pseudoElt);
+  external DomCSSStyleDeclaration _getComputedStyle(
+      DomElement elt, [JSString pseudoElt]);
   DomCSSStyleDeclaration getComputedStyle(DomElement elt, [String? pseudoElt]) {
     if (pseudoElt == null) {
-      return _getComputedStyle1(elt);
+      return _getComputedStyle(elt);
     } else {
-      return _getComputedStyle2(elt, pseudoElt.toJS);
+      return _getComputedStyle(elt, pseudoElt.toJS);
     }
   }
```

Comment:
Do we still need the trampoline? Is it different to pass null than to not pass anything to pseudoElt of the external method?

Reply:
Yep. We still do a `pseudoElt.toJS`, so it's needed until I address the second bulletpoint of "still left to do".


The semantics of calling an `external` interop method are that it passes the only the arguments that are given and ignores any defaults. If you call `_getComputedStyle(elt, null)` (assuming the types allow it), it lowers to a `getComputedStyle(elt, null)` call and not `getComputedStyle(elt)`. If you call `_getComputedStyle(elt)`, then you get a `getComputedStyle(elt)`. We have documentation here too finally: https://dart.dev/interop/js-interop/usage#parameters :D

This trampoline `getComputedStyle` is never called with `null` as its second argument, so it should be safe to just replace this with:

```dart
external DomCSSStyleDeclaration getComputedStyle(DomElement elt, [String pseudoElt]);
```

when I address the second bulletpoint.